code/game/machinery/autolathe.dm

```diff
@@ -174,7 +174,8 @@
 				return
 
 			var/multiplier = text2num(params["multiplier"])
-			if(!multiplier)
+			if(!multiplier || !isnum(multiplier) || ISNAN(multiplier))
+				message_admins("[ADMIN_FULLMONTY(usr)] is attempting to create NAN stacks of items")
```

Comment:
You might want to spell it out for the admins, they can be a little... you know.

Reply:
this is only for the test merge